src/es/carbon-intensity.md

```diff
@@ -1,63 +1,61 @@
 ---
-title: Carbon Intensity
-summary: Consume electricity with the lowest carbon intensity
+title: Intensidad de carbono
+summary: Consuma electricidad con la mínima intensidad de carbono
 order: 3
 tags: principle
 ---
 
-The carbon intensity of electricity is a measure of how much carbon (CO<sub>2</sub>eq) emissions are produced per kilowatt-hour of electricity consumed.
+La intensidad de carbono de la electricidad es una medida de cuantas emisiones de carbono (CO<sub>2</sub>eq) se producen por kilovatio hora de electricidad consumida.
 
-The standard unit of carbon intensity is gCO<sub>2</sub>eq/kWh or grams of carbon per kilowatt-hour.
+La unidad estándar de intensidad de carbono es gCO<sub>2</sub>eq/kWh o gramos de carbono equivalente por kilovatio hora.
 
-Not all electricity is produced in the same way. In different locations and at different times, the electricity is produced using a variety of sources with *different carbon emissions*. Some sources, such as wind, solar, or hydroelectric, are clean, *renewable sources* that emit no carbon. Other *fossil fuel sources* emit varying amounts of carbon to produce electricity. For example, gas-burning power plants emit less carbon than coal-burning power plants.
+No toda la electricidad se produce de la misma forma. En distintas ubicaciones a distintas horas, la electricidad se produce usando una variedad de fuentes con *distintas emisiones de carbono*. Algunas fuentes, como eólica, solar, o hidroeléctrica, son *fuentes de energía renovable* limpias que no emiten carbono. Otras *fuentes basadas en combustibles fósiles* emiten distintas cantidades de carbono para producir electricidad. Por ejemplo, las centrales termoeléctricas de gas emiten menos carbono que las centrales termoeléctricas de carbon.
 
-If your computer is plugged directly into a wind farm, then the electricity it consumes would have a carbon intensity of 0 gCO<sub>2</sub>eq/kWh since a wind farm emits no carbon to produce that electricity. Most people can’t plug directly into wind farms, they instead plug into power grids that are usually supplied with electricity from a mix of sources that produce varying amounts of carbon. Therefore, when plugged into a grid your carbon intensity is usually a number greater than 0. 
+Si su computadora está enchufada directamente a un parque eólico, entonces la electricidad que consume tendría una intensidad de carbono de 0 gCO<sub>2</sub>eq/kWh ya que un parque eólico no emite carbono para producir electricidad. La mayoría de gente no se pueden conectar directamente a un parque eólico, en su lugar se conectan a redes eléctricas que se suelen proveer de electricidad fe una mezcla de fuentes que producen distintas cantidades de carbono. Por lo tanto, cuando se conecta a una red su intensidad de carbono es normalmente un número mayor que 0.
```

Comment:
A minor typo here "se suelen proveer de electricidad **fe** una mezcla de fuentes" should be "se suelen proveer de electricidad **de** una mezcla de fuentes"

```suggestion
Si su computadora está enchufada directamente a un parque eólico, entonces la electricidad que consume tendría una intensidad de carbono de 0 gCO<sub>2</sub>eq/kWh ya que un parque eólico no emite carbono para producir electricidad. La mayoría de gente no se pueden conectar directamente a un parque eólico, en su lugar se conectan a redes eléctricas que se suelen proveer de electricidad de una mezcla de fuentes que producen distintas cantidades de carbono. Por lo tanto, cuando se conecta a una red su intensidad de carbono es normalmente un número mayor que 0.
```